includes/providers/gam/class-gam-api.php

```diff
@@ -1394,23 +1386,18 @@ private static function get_connection_details() {
 	 * @return object Object with status information.
 	 */
 	public static function connection_status() {
-		$connection_details      = self::get_connection_details();
-		$can_use_oauth           = self::can_use_oauth();
-		$can_use_service_account = self::can_use_service_account();
-		$response                = [
-			'connected'               => false,
-			'connection_mode'         => $connection_details['mode'],
-			'can_use_oauth'           => $can_use_oauth,
-			'can_use_service_account' => $can_use_service_account,
+		$connection_details = self::get_connection_details();
+		$can_use_oauth      = self::can_use_oauth();
+		$response           = [
+			'connected'       => false,
+			'connection_mode' => $connection_details['mode'],
+			'can_use_oauth'   => $can_use_oauth,
```

Comment:
Non-blocking: from what I can tell, we're no longer using this variable anywhere, so we can probably remove it from this response as well as its associated method.

Reply:
Good call! Done in e90c8194fc8c9e95d7d4b39631309b11e19dfb05, which also linted blank spaces due to my new editor settings 😬